packages/connection/src/class-manager.php

```diff
@@ -2139,7 +2139,7 @@ public function get_access_token( $user_id = false, $token_key = false, $suppres
 
 		if ( $user_id ) {
 			if ( ! $user_tokens ) {
-				return $suppress_errors ? false : new \WP_Error( 'no_user_tokens' );
+				return $suppress_errors ? false : new \WP_Error( 'no_user_tokens', 'No user tokens found' );
```

Comment:
Will these ever be displayed to end users? (do they need to be translated?)

Reply:
They are not currently displayed anywhere at all. We want to start displaying them in the Jetpack Debugger now.

I don't think we will use it to display to the user at any point, we will most likely continue what we are doing which is grouping many error codes in one single friendly user message. This detailed error code will serve just for debugging.

In any case, if we decide it's best to translate it, we will have to add i18n function in all error messages there. I just added messages to a few errors that didn't have one, but most of them have - and all of them are not internationalized.